pypeit/core/wavecal/wvutils.py

```diff
@@ -36,35 +37,55 @@ def parse_param(par, key, slit):
 
     return param
 
```

Comment:
I agree that a non-smooth option could be desirable, but that is as easy as np.diff in most cases. We could rename this to avoid confusion to "get_smooth_delta_wave". Anyway, the reason for the smoothing is that we need counts/Ang/s for sensitivity functions and that means dividing by delta_wave. Given how we extract, delta_wave could be jumpy, which is why I found smoothing was required.

Reply:
Okay, leaving it as is.